docs/_rpc/ns-debug.md

```diff
@@ -88,9 +113,33 @@ storage and code).
 }
 ```
 
+### debug_freeOSMemory
+
+Forces garbage collection
+
+| Client  | Method invocation                                 |
+|:--------|---------------------------------------------------|
+| Go      | `debug.freeOSMemory()`                            |
+| Console | `debug.freeOSMemory()`                            |
+| RPC     | `{"method": "debug_freeOSMemory", "params": []}`  |
+
+
```

Comment:
Sorry this is a big bikeshed thing but that seems like an excessive number of empty lines :d I usually stay consistently with one line between everything. Makes keyboard navigation easier.